packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -272,22 +272,26 @@ export const ConnectedServices = () => {
               </Localized>
 
               <Localized
-                id="cs-disconnect-sync-content"
+                id="cs-disconnect-sync-content-2"
                 vars={{ device: selectedClient!.name }}
               >
                 <p
                   id="connected-devices-sign-out-description"
                   className="my-4 text-center"
                 >
-                  Your browsing data will remain on your device (
-                  <span className="break-word">{selectedClient!.name}</span>
-                  ), but it will no longer sync with your account.
+                  Your browsing data will remain on
```

Comment:
We'll need to add a space here, otherwise the device name gets concatenated with "on".